apps/morph_stats

```diff
@@ -93,12 +93,35 @@ def _get_config(config):
         return _CONFIG
 
 
-if __name__ == '__main__':
-    args = parse_args()
-    logging.basicConfig(level=(logging.WARNING,
-                               logging.INFO,
-                               logging.DEBUG)[min(args.verbose, 2)])
-
+def _get_header(results):
```

Comment:
Totally agree.

I'll move it there when I get some time.  Thanks for pointing out where it should go.